hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -2649,4 +2653,42 @@ public String toString() {
                 + ", metricsConfig=" + metricsConfig
                 + '}';
     }
+
+    /**
+     * Populate Hazelcast <code>Config</code> object from an external configuration file.
+     * It tries to load Hazelcast configuration from a list of well-known locations.
+     * When no location contains Hazelcast configuration then it returns default.
+     *
```

Comment:
Can you add a note that the same mechanism is used when you call Hazelcast.newHazelcastInstance() ?